K8s-deployment/databroker/deploy-dependencies.sh

```diff
@@ -0,0 +1,11 @@
+kubectl apply -f namespace.yaml
+kubectl create secret generic certs --from-file=./secrets/pki/rabbitmq-ca-cert.pem --from-file=./secrets/pki/rabbitmq-server-cert.pem --from-file=./secrets/pki/rabbitmq-server-key.pem -n rabbitmq
+kubectl create secret generic rabbitmq-erlang-cookie --from-file=./secrets/credentials/.erlang.cookie -n rabbitmq
```

Comment:
'secrets/credentials/.erlang.cookie' should add this file in the secrets directory structure in the README